### PR TITLE
docs: fix typos and description

### PR DESCRIPTION
**Pull request by pdecat**
- docs(typo): Fix TDP => TCP
- docs(acm): Fix title of ACM certificates expiration control

### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: conformance_pack/acm.sp [hunk 1]
```diff
@@ -5,7 +5,7 @@ locals {
 }
 
 control "acm_certificate_expires_30_days" {
-  title       = "ACM certificates should be set to expire within 30 days"
+  title       = "ACM certificates should not expire within 30 days"
```
Comment (pdecat) [inline conformance_pack/acm.sp:8]: That's what the query does
https://github.com/turbot/steampipe-mod-aws-compliance/pull/719/files#diff-d695094acef9ce7b8cac5c5ed4ba873d9d94ee396aa011afab6965a21b42a767L52-L53

Comment (rajlearner17) [reply conformance_pack/acm.sp:8]: @pdecat - We derived this from [AWS Config](https://docs.aws.amazon.com/config/latest/developerguide/acm-certificate-expiration-check.html) rule, the language was adjusted based on its description. 
IMO, AWS has lowered further to 14 days as default from 30 days; we need to revalidate and adjust this query.

CC @khushboo9024

Comment (pdecat) [reply conformance_pack/acm.sp:8]: I guess it is just a matter of wording then.
Given the original description:
>  The rule is NON_COMPLIANT if your certificates are about to expire.

What about :
```suggestion
  title       = "ACM certificates will expire within 30 days"
```

Comment (cbruno10) [reply conformance_pack/acm.sp:8]: Usually our titles describe the OK/good state, e.g., CloudFormation stacks outputs should not have any secrets, SQS queues should be configured with a dead-letter queue. So in this case, I think "ACM certificates should not expire within 30 days" is a better title.

**Review by cbruno10 — APPROVED** [review]

**Comment by cbruno10** [comment]
Thanks @pdecat for the fixes! We'll release this in our next version most likely sometime next week.

**Pull request merged**